src/main/java/net/minestom/server/extensions/DiscoveredExtension.java

```diff
@@ -81,7 +120,27 @@ public File getOriginalJar() {
         return originalJar;
     }
 
-    static void verifyIntegrity(@NotNull DiscoveredExtension extension) {
+    public MinestomExtensionClassLoader removeMinestomExtensionClassLoader() {
+        MinestomExtensionClassLoader oldClassLoader = getMinestomExtensionClassLoader();
+        setMinestomExtensionClassLoader(null);
+        return oldClassLoader;
+    }
+
+    public void setMinestomExtensionClassLoader(@Nullable MinestomExtensionClassLoader minestomExtensionClassLoader) {
+        this.minestomExtensionClassLoader = minestomExtensionClassLoader;
+    }
```

Comment:
Why make these functions public? That's opening a way for users to break how extension classloaders work, and someone will complain some day.